pkgs/stdenv/generic/check-meta.nix

```diff
@@ -136,6 +144,14 @@ let
     '';
 
   remediate_insecure = attrs:
+    let
```

Comment:
One last little note: can we move this one or more levels up to dedupe it?

Other than that LGTM

Reply:
Done.